packages/next-tinacms-github/README.md

```diff
@@ -65,7 +65,7 @@ Below is an example of the conditional data fetching, from the local environment
 
 ```ts
 import {
-  getGithubPreviewProps,
+  getGithubPreviewProps
```

Comment:
Was this deletion intentional?

Reply:
> o

I got more work coming, I'm nuking this whole package